spring-kafka-test/src/main/java/org/springframework/kafka/test/EmbeddedKafkaBroker.java

```diff
@@ -115,6 +119,27 @@
 
 	public static final int DEFAULT_ZK_SESSION_TIMEOUT = 6000;
 
+	private static final String clientVersion;
```

Comment:
Doesn't look like we need this as a property: you just use a version info in the exception message.